### ISSUE TITLE
[Bug]: CryptographyDeprecationWarning with certbot 3.0.0

### ISSUE THREAD

**Issue by Tassleh0ff** [infinityofspace/certbot_dns_duckdns#155]
### OS info

Debian 12 aarch64

### Docker image

- [X] I am using the docker image.

### certbot_dns_duckdns version

v1.4
digest: sha256:93fccf5ed36ee6f768e1d4d91f16896e71d8e9dc6719d6779b5d6a96f15ceb08

### certbot version

3.0.0

### Bug description

Launching certbot with the 1.4 version (running certbot 3.0.0) results in these warnings
```
/opt/venv/lib/python3.13/site-packages/certbot/ocsp.py:238: CryptographyDeprecationWarning: Properties that return a naïve datetime object have been deprecated. Please switch to this_update_utc.
  if not response_ocsp.this_update:
/opt/venv/lib/python3.13/site-packages/certbot/ocsp.py:240: CryptographyDeprecationWarning: Properties that return a naïve datetime object have been deprecated. Please switch to this_update_utc.
  if response_ocsp.this_update > now + timedelta(minutes=5):
/opt/venv/lib/python3.13/site-packages/certbot/ocsp.py:242: CryptographyDeprecationWarning: Properties that return a naïve datetime object have been deprecated. Please switch to next_update_utc.
  if response_ocsp.next_update and response_ocsp.next_update < now - timedelta(minutes=5):
```

### Expected behaviour description

No CryptographyDeprecationWarning warnings on certbot output

### Command

```shell
docker run --rm -v /etc/letsencrypt:/etc/letsencrypt infinityofspace/certbot_dns_duckdns:v1.4 "certificates"
```

### Steps to reproduce

docker run --rm -v /etc/letsencrypt:/etc/letsencrypt infinityofspace/certbot_dns_duckdns:v1.4 "certificates"

### Additional context

This bug has been reported on Certbot Github project in [this issue](https://github.com/certbot/certbot/issues/9967).

A workaround has been reported setting the env variable PYTHONWARNINGS=ignore
For example the same command with the PYTHONWARNINGS=ignore env variable do not return the warnings
```
docker run --rm -v /etc/letsencrypt:/etc/letsencrypt -e PYTHONWARNINGS=ignore infinityofspace/certbot_dns_duckdns:v1.4 "certificates"
```

### Checklist

- [X] I have checked for [existing Github issues](https://github.com/infinityofspace/certbot_dns_duckdns/issues) for the same bug.
- [X] I have checked to see if there is a [newer version](https://github.com/infinityofspace/certbot_dns_duckdns/releases) that already fixes this error.

**Comment by infinityofspace** [comment]
Thanks for your bug report and hints for a workaround. The problem has already been fixed in certbot version [`3.0.1`](https://github.com/certbot/certbot/releases/tag/v3.0.1) and will be fixed with the upcoming release of certbot_dns_duckdns version `v1.5`.

**Comment by infinityofspace** [comment]
The new docker image with version `v1.5` is now available and should fix this issue.

Feel free to reopen the issue if the issue still exists.
